LICENSE.md

```diff
@@ -1,5 +1,5 @@
 The Code:
-The Spring Engine (which this game runs on) requires that all the code the games use are under the GPL licence. (https://springrts.com)
+The Spring Engine (which this game runs on) requires that all the code the games use are under the GPL v2 licence. (https://springrts.com)
```

Comment:
It only requires the code to be compatible with v2 or later. v3 is fine, MIT is fine, etc